<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

  <include file="$(find melle_obstacle_avoidance)/launch/melle_ob_av.launch"/>
  <include file="$(find downview_cam)/launch/downview_cam.launch"/>
  <node name="pc_node" pkg="melle_refactored" type="melle_refactored">
  </node>

  <node respawn="true" pkg="joy"
        type="joy_node" name="joy_node" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.2" />
  </node>
  
  <!-- Startup your modified apriltag wrapper node -->
  <!-- TODO -->
  <node name="serial_node" pkg="rosserial_python" type="serial_node.py">
  <param name="~port" value="/dev/ttyACM0" />
  </node>

</launch>